Rajitha/Palindrome.js

```diff
@@ -0,0 +1,43 @@
+/***************************************************
+ * Determine whether a given input is a palindrome.
+ ***************************************************/
+const palindrome = inputQuery => {
+  var query = '';
+  var count = 0;
+  
+  //If number convert to string
+  if (typeof inputQuery === "number"){
+    inputQuery = inputQuery.toString();
+  }
+  count = inputQuery.length;
+
+  //Convert string to lower case and remove special characters/empty spaces
+  inputQuery = inputQuery.toLowerCase().replace(/\W|_/g, "");
+  
+  //Iterate and store characters starting from end
+  //Compare final value with input
+  for (var i = count-1; i >= 0; i--){
+    query += inputQuery.charAt(i);
+  }
+  return((query === inputQuery)? true : false);
```

Comment:
good idea, but this can be refactored. Creating `query` adds unnecessary space. Think of what you really need without additional items. 

what is the difference between `return((query === inputQuery)? true : false);` and `return query === inputQuery`?